pages/index.js

```diff
@@ -32,7 +32,7 @@ function Home () {
 			/>
 			<div className="home-page text-center">
 				<ContentBanner
-					title="ACM's Winter General Meeting is coming soon!"
+					title="Check out our Winter General Meeting!"
```

Comment:
Might want to reword?

`See what happened?` something more indicative of the past?

Reply:
Oh yeah that's a good idea